exchange/exchangetest/first-party-data-multiple-bidders.json

```diff
@@ -0,0 +1,173 @@
+{
+    "incomingRequest": {
+        "ortbRequest": {
+            "id": "some-request-id",
+            "site": {
+                "page": "test.somepage.com"
+            },
+            "imp": [{
+                "id": "some-imp-id",
+                "banner": {
+                    "format": [{
+                        "w": 600,
+                        "h": 500
+                    }, {
+                        "w": 300,
+                        "h": 600
+                    }]
+                },
+                "ext": {
+                    "appnexus": {
+                        "placementId": 1
+                    },
+                    "rubicon": {
+                        "accountId": 1,
+                        "siteId": 2,
+                        "zoneId": 3
+                    },
+                    "context": {
+                        "data": {
+                            "keywords": "prebid server example"
+                        }
+                    }
+                }
+            }]
+        }
+    },
+    "outgoingRequests": {
+        "appnexus": {
+            "expectRequest": {
+                "ortbRequest": {
+                    "id": "some-request-id",
+                    "site": {
+                        "page": "test.somepage.com"
+                    },
+                    "imp": [{
+                        "id": "some-imp-id",
+                        "banner": {
+                            "format": [{
+                                "w": 600,
+                                "h": 500
+                            }, {
+                                "w": 300,
+                                "h": 600
+                            }]
+                        },
+                        "ext": {
+                            "bidder": {
+                                "placementId": 1
+                            },
+                            "context": {
```

Comment:
This is considered a bidder today and causes all sorts of panics currently (when the validation now allows it).